prompt_engineering/prompt_engineering.py

```diff
@@ -42,7 +41,7 @@ def read_pdf(file_path):
     return text
 
 
-def clean_pdf_text(text):
+def clean_text(text):
     out_text = text
     for word in ['References', 'references', 'References'.upper()]:
         index = text.rfind(word)
```

Comment:
1. Since you are doing exactly the same thing for two concepts that could be united to one operation with different keywords

Reply:
done